### PR TITLE
Updates for Making a one-time payment

### PR DESCRIPTION
**Pull request by bwentwor**
This change corrects the "Making a one-time payment" section and adds information about when one-time payments are processed.

These changes are needed for the cloud assistant.

### CODE REVIEW

File: manage_billing.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Some payment methods, such as PayPal, aren't accepted as recurring payment metho
...
 
-To make a one-time payment, in the {{site.data.keyword.cloud_notm}} console, go to **Manage > Billing and usage**, and select **Payments**. Complete the fields in the make a one-time payment section. The account balance is updated after the payment is accepted.
+To make a one-time payment, in the {{site.data.keyword.cloud_notm}} console, go to **Manage > Billing and usage**, and select **Payments**. Click **Manual Payment** and complete the fields in the Make A One Time Payment section. One-time payments are reviewed and processed throughout each day. The account balance is updated after the payment is accepted. If a late fee is assessed and you have submitted a one-time payment, contact  [{{site.data.keyword.cloud_notm}} Support](https://cloud.ibm.com/unifiedsupport/supportcenter) {: external} for assistance.
```
Comment (hooznora) [inline manage_billing.md:85]: This section looks good!

Comment (bwentwor) [reply manage_billing.md:85]: Please commit these changes.

**Comment by bwentwor** [comment]
@hooznora This change is live at https://cloud.ibm.com/docs/billing-usage?topic=billing-usage-linkedusage
Can you close this issue?

**Comment by hlucey** [comment]
Confirmed with @hooznora that this was implemented.

**Pull request closed without merging**